docs/getting-started/tutorials/backoffice/auth0-integration.mdx

```diff
@@ -0,0 +1,756 @@
+---
+id: backoffice-auth0-integration
+title: Backoffice Integration with Auth0
+sidebar_label: Backoffice Integration with Auth0
+---
+
+# Mia-Platform Backoffice Integration with Auth0
+
+In this tutorial, we will integrate the Backoffice with Auth0. This will allow us to:
+- create, delete and manage Auth0 users using the Backoffice;
+- create, delete and manage users' groups to handle the access to Backoffice pages using ACLs;
+- setup a frontend where users can get access to the Backoffice.
+
+## What We Will Build
+
+Throughout this tutorial, we will:
+
+- [Set Up the Microservices](#setup-the-microservices)
+- [Configure the Microservices and Auth0](#configure-the-microservices-and-auth0)
+- [Configure the Backoffice](#configure-the-backoffice)
+- [Configure the authentication process](#configure-the-authentication-process)
+- [Save and Deploy the Backoffice](#save-and-deploy)
+- [Create a user using Backoffice](#create-a-user-using-backoffice)
+
+## Prerequisites
+
+Before starting this tutorial, we need to be familiar with the concepts of the [Mia-Platform Backoffice](/business_suite/backoffice/10_overview.md) and have completed:
+* The [Backoffice Tutorial Basics](/getting-started/tutorials/backoffice/basics.mdx).
+
+This tutorial will use the following application:
+* [Secure API Gateway](../../../runtime_suite_applications/secure-apigateway/overview)
+
+and some of the concepts described on this tutorial:
+* [Dev Portal - Authentication Configuration](../../../runtime_suite_applications/dev_portal/authentication_configuration)
+
+## Setup the Microservices
+
+You can install almost all the microservices for this tutorial using the [Secure API Gateway](../../../runtime_suite_applications/secure-apigateway/overview) application:
+1. Go to Applications and click on the `Create New Application` button.
+2. Search for `Secure ApiGateway` and click on the card.
+![Secure ApiGateway Application Marketplace](img/auth0_01-secure-api-gateway-application.png)
+3. Follow the wizard, selecting if you want to install new microservices or use existing ones.
+4. At the end, click the `Create` button and then save the new Application.
+
+This application will create the following:
+* Microservices 
```

Comment:
I would say somehow that these are MIa-Platform plugins. They do not require the user to write code but just the configuration

Reply:
I've changed the sentence under the **Configure the Microservices and Auth0** section with the following:

> All the microservices that you have added in the previous step are Mia-Platform plugins, so you don't need to write code, but just to configure them.